client/my-sites/upgrades/checkout/privacy-protection.jsx

```diff
@@ -59,22 +59,27 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
-		var numberOfDomainRegistrations = this.getNumberOfDomainRegistrations(),
-			priceForPrivacyText,
+		const numberOfDomainRegistrations = this.getNumberOfDomainRegistrations(),
 			hasOneFreePrivacy = this.hasDomainPartOfPlan() && numberOfDomainRegistrations === 1;
+		let priceForPrivacyText,
```

Comment:
It seems to me that all of these `let` variables are actually constant (they are assigned one/the same value, just below) and could be moved to the `const` block above.